setup.py

```diff
@@ -35,6 +35,7 @@ def read(*parts):
             'datasource_health = redash_stmo.data_sources.health:health',
             'datasource_link = redash_stmo.data_sources.link:link',
             'datasource_version = redash_stmo.data_sources.version:version',
+            'query_results_permissions = redash_stmo.handlers.query_results:query_results_permissions',
```

Comment:
I think it would be nice to have consistent entrypoint names, e.g. `handler_queryresult`, what do you think?